doc/customization_basic1.rst

```diff
@@ -0,0 +1,120 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _customization_basic1:
+
+#######################
+2. Basic customization
+#######################
+
+The goal of this section is to show you how to make basic settings to customize the configuration file and consequently the execution of the tool.
+First, it is worth spending a few words on the configuration file.
+Within the work folder there is always a configuration file ( typically the file that is read by snakemake is the "config.yaml).
+This type of file is used to define and customize the settings and parameters for the execution of the workflow or process, so any customization of the tool will be performed on that file. 
+In the next steps we will see the basic settings in order to customize the configuration file, in case you are interested in more in-depth customization, you can refer to the configuration section in the Pypsa-Earth documentation.
+
+
+Specify the country/region of interest
+--------------------------------------
+
+The model can be adapted to include any country, multiple countries (e.g. `Nigeria` and `Benin`)  using ``countries`` argument:
+
+.. code:: yaml
+
+    countries: ["NG"]
+
```

Comment:
I see this is the copy and paste from pypsa-earth; have you tried to follow the procedure to see if it works for -distribution?

In the short tutorial, if tutorial=true, only NG,BJ,BW and MA are supported. this should be mentioned.
Moreover, there is the need to specify the microgrid to consider using the microgrid_list  option

Probably, it may be good to have this customization generally applicable to any country; in that case we should specify to make sure tutorial=false in config.yaml

Reply:
_What do you think if i change this section in this way? ( I have tried to write it in such a way that as soon as the tool is working for all countries it will be sufficient to make minor changes to the text )_


Specify the country of interest: 
--------------------------------------

Pypsa-Distribution is a tool under development, how soon it will be possible to use it for the study and simulation of microgrids in all countries.

Selection of the country of interest will be possible using the "coutries" argument at the top of the config file.

.. code:: yaml
    countries: ["NG"]


Please note,before proceeding, a premise must be made: in analogy with Pypsa-Earth, the simulation will be available in "tutorial" mode and "full" mode. 

The tutorial mode is selectable by placing within the config file:

. code:: yaml

    tutorial: true

It allows for faster simulation with less data, so that new users can become familiar with the code without running overly heavy simulations. As with Pypsa-Earth, simulation in tutorial mode will be made possible for a small number of countries ( Nigeria, Benin, Botswana, and Morocco).

To perform simulation of other countries it will be necessary to set:

. code:: yaml

    tutorial: false

And run a full simulation.

At the current state of development Pypsa-Distribution is able to run simulations in tutorial mode on Nigeria, but the potential of the tool will be extended as soon as possible.

Reply:
Ok, but at the line "Selection of the country of interest will be possible using the "coutries" argument at the top of the config file.", I'd add "For the quick tutorial, "NG, BW, BJ and MA" are supported with the tutorial mode; for the full model or other region, please refer to {...REF}"